AirSkyBoat/AirSkyBoat
# Updates Changelogs and corrects scripts to generate the correct date

**_I affirm:_**


## Please enter a player-facing description
Nothing to say here


## What does this pull request do? (Please be technical)
Updates Changelogs and corrects scripts to generate the correct date.


## Steps to test these changes
Run this command and see that it calculates the days correctly:
py .\generate_changelog.py ci AirSkyBoat/AirSkyBoat


## Special Deployment Considerations
Nothing to say here
